sql/wsrep_mysqld.cc

```diff
@@ -2845,7 +2845,16 @@ static int wsrep_RSU_begin(THD *thd, const char *, const char *) {
   if (thd->wsrep_cs().begin_rsu(wsrep_RSU_commit_timeout)) {
     WSREP_WARN("RSU begin failed");
   } else {
-    thd->variables.wsrep_on = 0;
+    WSREP_WARN(
+        "DDL executed with wsrep_OSU_method = RSU."
+        " Transaction not binlogged, GTID not generated."
+        " Query: %s",
+        WSREP_QUERY(thd));
+    push_warning_printf(thd, Sql_condition::SL_WARNING, ER_UNKNOWN_ERROR,
+                        "DDL executed with wsrep_OSU_method = RSU."
+                        " Transaction not binlogged, GTID not generated.");
```

Comment:
Can the warning be something like?
```suggestion
                        "The statement was executed with wsrep_OSU_method = RSU."
                        "Binary log will be disabled during the execution of the current "
                        "statement and the statement will not be written to the binlog");
```

Reply:
done